guidelines/enhancement_template.md

```diff
@@ -72,17 +72,33 @@ this proposal. Describe why the change is important and the benefits to users.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is
-implemented. In each story, explain who the actor is based on their
-role, explain what they want to do with the system, and explain the
-underlying goal they have. Use the standard three part formula:
+Detail the things that people will be able to do if this is implemented and
+what goal that allows them to achieve .In each story, explain who the actor
+is based on their role, explain what they want to do with the system,
+and explain the underlying goal they have, what it is they are going to
+achieve with this new feature.
 
-"As a <role>, I want to <take some action> so that I can <accomplish a
-goal>."
+Use the standard three part formula:
+
+"As a \<role\>, I want to \<take some action\> so that I can \<accomplish a
+goal\>."
 
 Make the change feel real for users, without getting bogged down in
 implementation details.
 
+Here are some example user stories to show what they might look like:
+
+* As an OpenShift engineer, I want to write an enhancement, so that I can share my feature idea
+and get feedback from others.
```

Comment:
Nobody _wants_ to write an enhancement ;).  Maybe:

> As an OpenShift engineer, I want to pitch a possible feature implementation to my colleagues, so we don't end up swamped in technical-debt like some of the alternative implementations for this RFE.

or something?

Reply:
We're specifically telling people not to pitch via enhancements.

"As an OpenShift engineer, I want to write an enhancement, so that I can get feedback on my design and build consensus about the approach to take before starting the implementation."